src/topology/sheaves/sheaf_condition/basis_le.lean

```diff
@@ -0,0 +1,273 @@
+/-
+Copyright (c) 2021 Junyan Xu. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Junyan Xu
+-/
+
+import topology.sheaves.sheaf_condition.opens_le_cover
+
+/-!
+# Consequences of the sheaf condition about covers by open sets in a topological basis
+
+Let `B` be a basis of a topological space `X`.
+
+This file shows that if `G` is a sheaf on `X`,
+then `G(U)` is the limit of all `G(V)` with `V ∈ B, V ⊆ U`,
+and if `α : F ⟶ G` is a homomorphism of presheaves on `X`
+defined only on open sets in `B`, it uniquely extends to an
+actual homomorphism on all open sets.
+
+-/
+
+universes v u
+
+noncomputable theory
+
+open category_theory
+open category_theory.limits
+open topological_space
+open opposite
+open topological_space.opens
+
+namespace Top
+
+variables {C : Type u} [category.{v} C] (X : Top.{v})
+
+/-- Structure for indexing a family of open sets in a topological space. -/
+structure opens_index_struct :=
```

Comment:
I don't think this is a good structure to make here. Everywhere else, we always take in a family of open sets as two arguments: `ι` and `f`. And if we were to define bundled families of open sets, the `sheaves/sheaf_condition` subdirectory is certainly not the right place.

Reply:
Bundled version seems necessary for Lean to infer the induced category structure on `ι`; I recall that I seem not be able to make explicitly provided instances work. I agree it's not the right place to define it.

Reply:
No longer applicable after refactoring.